Lab_WebDriverAdvanced. Загрузка

UlyanaRomanova opened the pull request:
Я не смогла победить обработку окна предупреждения в allProductsPage.RemoveTestProduct() через Actions. 
Оставила через driver.

UlyanaRomanova committed:
Lab_WebDriverAdvanced. Загрузка

DuskTraum commented:
- Не оставляй закоментированный код
- AllProductsPage, CreateProductPage лучше параметризировать локаторы для работы с продуктом, а не хардкодать их
-  в GetSubmitButtonSelector() используй уже найденный элемент, а не локатор

UlyanaRomanova committed:
Исправление замечаний.

немного исправила IsElementPresent(). Теперь он принимает элемент. Соответственно и методы, которые возвращали локаторы, теперь возвращают элементы.

DuskTraum commented:
Принято
